src/stdune/list.ml

```diff
@@ -15,10 +15,11 @@ let is_non_empty = function
 let filter_map l ~f =
   let rec loop acc = function
     | [] -> rev acc
-    | x :: xs -> (
-      match f x with
+    | x :: xs ->
+      begin match f x with
```

Comment:
why was this changed?

Reply:
Sorry for the late reply. That file wasn't well formatted at the previous base of this branch (more than a month ago)